packages/lib-react-components/.storybook/preview.js

```diff
@@ -16,3 +15,36 @@ export const parameters = {
     viewports: INITIAL_VIEWPORTS
   }
 }
+
+export const globalTypes = {
+  theme: {
+    name: 'Grommet Theme',
+    description: 'Global Grommet theme for components',
+    defaultValue: 'light',
+    toolbar: {
+      icon: 'arrowdown',
+      items: ['light', 'dark'],
+      showName: true,
+    }
+  }
+}
+
+const background = {
+  dark: 'dark-1',
+  light: 'light-1'
+}
+
+export const decorators = [
```

Comment:
[note to self][dev notes] Note the `export` here, as this defines the _global_ `decorators` _array_ (see [docs](https://storybook.js.org/docs/react/writing-stories/decorators#global-decorators)) that automagically wraps all Stories in this Package.

[note to self][wild ramblings] Surprisingly, unlike the docs, there's no instance of `import { Preview } from '@storybook/react' ; preview = Preview... ; export default preview` here, so I guess Storybook Stories just do something like `import { decorators } from 'preview.js'` instead of `import preview from '.storybook/preview.js' ; preview.decorators`? It still works, but I wonder what that Preview object was for.